packages/api-server/src/app/app.ts

```diff
@@ -16,6 +17,17 @@ if (envConfig.newRelicLicenseKey) {
   newrelic = require("newrelic");
 }
 
+//TODO: maybe config init to cluster master, only run once
+// init godwoken config
+gwConfig.init(
+  () => {
+    logger.info("godwoken config initialized!");
+  },
+  (error) => {
```

Comment:
Shall we retry on error? Or exit and let docker/k8s restart us?

Reply:
agree, that's better

Reply:
I check and found that to exit the process here will require some cluster code refactor, I will update in another pr.